commons-web/src/main/java/eu/europeana/api/commons/web/model/vocabulary/Operations.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.api.commons.web.model.vocabulary;
+
+public interface Operations {
+
+	//search is also retrieve
+	//crud operations
+	public static final String RETRIEVE = "retrieve";
+	public static final String CREATE = "create";
+	public static final String UPDATE = "update";
+	public static final String DELETE = "delete";
+	
+	//admin
+	public static final String ADMIN_ALL = "admin_all";
+	public static final String ADMIN_UNLOCK = "admin_unlock";
```

Comment:
only the CRUD and ADMIN_ALL are generic permission to stay within the api-commons modules. The rest of them are api specific and must not be included here..

Reply:
api specific permissions removed from the file